modin/pandas/__init__.py

```diff
@@ -226,6 +226,14 @@ def move_stdlib_ahead_of_site_packages(*args):
             move_stdlib_ahead_of_site_packages
         )
 
+        # Register a fix to import pandas on all workers before running tasks.
+        # This prevents a race condition between two threads deserializing functions
+        # and trying to import pandas at the same time.
+        def import_pandas(*args):
+            import pandas
```

Comment:
Can you please change the line like this to tell flake8 lint to ignore the error?
```python
    import pandas  # noqa F401
````
two spaces between code and `#` symbol are required for this to work.